Resources/Public/Javascript/PageView/FullTextUtility.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+/**
+ * (c) Kitodo. Key to digital objects e.V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo and TYPO3 projects.
+ *
+ * @license GNU General Public License version 3 or later.
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ */
+
+ /**
+ * Base namespace for utility functions used by the dlf module.
+ *
+ * @const
+ */
+const dlfFullTextUtils = dlfFullTextUtils || {};
```

Comment:
@sebastian-meyer Do you have some idea how this Codacy issue can be fixed?

Reply:
Try using `var` instead of `const`. At least that's what I've seen others do. But I am not a JS expert myself. ;o)

Reply:
`var` was there at first, I changed it to try to fix it ;) I will try to first declare it and next assign it to itself...

Reply:
Well, that worked. ;)